### ISSUE TITLE
Create getting started guide for nim-libp2p

### ISSUE THREAD

**Issue by DannyS03** [libp2p/docs#245]
## Done Criteria
- [ ] Tutorial introduction: Ping application
- [ ] Prerequisites
  - [ ] Install Nim
  - [ ] Create an empty project
  - [ ] Install nim-libp2p
- [ ] Setup
  - [ ] Introduce [chronos](https://github.com/status-im/nim-chronos).
  - [ ] Create new file "test.nim"
  - [ ] Import dependencies
    - [ ] Import chronos
    - [ ] Import libp2p and ping protocol
- [ ] libp2p procedures
    - [ ] Gentle introduction to nim procedures
    - [ ] Create ping procedure
- [ ] Run the ping protocol

## Why Important
- Showcases how users can get started with nim-libp2p.
- Follows the other getting started tutorials on the docs.

## Notes
- Link to status.im tutorials and existing examples.

**Comment by p-shahi** [comment]
Ideally this is something the nim-libp2p maintainers can take on and not the PL libp2p or docs team

**Comment by Menduist** [comment]
Very cool, thanks!
We have a tutorial that looks quite similar to this on our website: https://status-im.github.io/nim-libp2p/docs/tutorial_1_connect/
Source is here: https://github.com/status-im/nim-libp2p/blob/unstable/examples/tutorial_1_connect.nim

The nice thing about having it on our repo is that we can keep it up to date more frequently when we add new sugar for instance, and make sure that it still compiles. I think rust is in the same situation as us (cc @mxinden).
Having it in another repo makes it a bit harder (I have PTSD from outdated examples & tutorials :D )

But maybe we can find a way to make it work, for instance some CI action that would keep it in sync here?

Also happy to get some feedback / fixes to our tutorial!

**Comment by mxinden** [comment]
>  Having it in another repo makes it a bit harder (I have PTSD from outdated examples & tutorials :D )

Agreed. I think example code should live as close to the code it is showcasing as possible.
